e2etests/orchestration/common/common.go

```diff
@@ -41,12 +39,16 @@ func Setup(port int) (*TestContext, error) {
 		return nil, err
 	}
 	result.DockerImageName = img
-	id, err := dockerHelper.RunContainer(img, port)
+	id, err := dockerHelper.RunContainer(img)
+	if err != nil {
+		return nil, err
+	}
+	ipAddr, err := dockerHelper.getIpAddr(id)
 	if err != nil {
 		return nil, err
 	}
 	result.DockerContainerID = id
-	result.ServiceURL = fmt.Sprintf("http://0.0.0.0:%d", port)
+	result.ServiceURL = fmt.Sprintf("http://%s:2080", ipAddr)
```

Comment:
can 2080 not be hardcoded?

Reply:
The docker image used in the e2e tests have the HO running at 2080 and we don't have logic in place to update the container and change the HO port. 

I used a const for the value though.